scripts/exglobal_atmos_sfcanl.sh

```diff
@@ -167,9 +167,14 @@ if [[ ${DOIAU} = "YES" ]]; then
     # update surface restarts at the beginning of the window, if IAU
     # For now assume/hold dtfanl.nc valid at beginning of window
     for n in $(seq 1 ${ntiles}); do
-        ${NCP} "${COM_ATMOS_RESTART_PREV}/${bPDY}.${bcyc}0000.sfc_data.tile${n}.nc" \
-                "${COM_ATMOS_RESTART}/${bPDY}.${bcyc}0000.sfcanl_data.tile${n}.nc"
-        ${NLN} "${COM_ATMOS_RESTART_PREV}/${bPDY}.${bcyc}0000.sfc_data.tile${n}.nc" "${DATA}/fnbgsi.00${n}"
+        if [[ ${DO_JEDISNOWDA:-"NO"} = "YES" ]]; then
+            src_dir="${COM_SNOW_ANALYSIS}"
```

Comment:
This `src_dir` definition should be moved to out of the `if [[ ${DOIAU} = "YES" ]]; then` block, because the `src_dir` will not be defined if the `DOIAU=NO`.